demos/smart_framing_demo/cpp_gapi/include/custom_kernels.hpp

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <opencv2/gapi/gkernel.hpp>
+#include <opencv2/gapi/infer/ie.hpp>
+
+#include <inference_engine.hpp>
+
+namespace IE = InferenceEngine;
+
+namespace custom {
+
+const std::vector<std::string> coco_classes = {
```

Comment:
I think you can load this from omz/data/dataset_calsses/coco_80cl.txt

Reply:
@mpashchenkov are aware about any examples of doing so for classes?

Reply:
FYI `omz/demos/instance_segmentation_demo `

I think you can just add `--labels` flag for path to file with labels.